### PR TITLE
bpo-31040: Add warning note about mimetypes.add_type ext argument format

### PR DESCRIPTION
**Pull request by OddBloke**
This addresses the documentation of the issue described in bpo-31040,
but not the underlying issue.

https://bugs.python.org/issue31040

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by OddBloke** [comment]
(I signed the CLA just before submitting this PR, so that should be cleared up fairly soon.)

File: Doc/library/mimetypes.rst [hunk 1]
```diff
@@ -109,6 +109,11 @@ behavior of the module.
    When *strict* is ``True`` (the default), the mapping will be added to the
    official MIME types, otherwise to the non-standard ones.
 
+   .. note::
+
+        The *ext* argument should start with a dot ("``.``") for the
```
Comment (JulienPalard) [inline Doc/library/mimetypes.rst:123]: Please also explicitly tell it can be empty, see b.p.o.

**Review by JulienPalard — CHANGES_REQUESTED** [review]

**Comment by csabella** [comment]
@OddBloke, please address the code review comments from @JulienPalard.  Thanks!

**Comment by csabella** [comment]
@OddBloke ping

**Comment by JelleZijlstra** [comment]
Closing as it's been almost two years since the requested change.

**Pull request closed without merging**